openforis/collect-earth-online
# CEO-21 Allow users to shuffle plot order for gridded and random

## Purpose
This is primarily for gridded plots where some users what the collection order to be random.  For random strata, the data is currently collected in order of strata, random in each.  This will allow all strata to be randomized together.

## Submission Checklist

## Testing
1. Create a project with gridded plots
2. Check the Shuffle Plot Order box
3. Collect to verify that the plots are in a random order. That is, when you zoom out while collecting on each of the first few plots, they aren't directly next to each other. You can also download the plot data CSV to verify that the lon and lat aren't in order.

## Screenshots
![image](https://user-images.githubusercontent.com/33734037/153288906-fb731842-9c3b-4c83-a566-230e09f718e8.png)
